workflows/proteomics/metaproteomics-gtn/metaproteomics_gtn_wf.ga

```diff
@@ -0,0 +1,724 @@
fix\": \"35\"}, \"save_db\": true, \"sqlquery\": \"SELECT sequence as \\\"peptide\\\", count(id) as \\\"PSMs\\\"\\nFROM bering_psms\\nWHERE confidence >= 95\\nGROUP BY sequence\\nORDER BY sequence\\n\", \"tables\": [{\"__index__\": 0, \"table\": {\"__class__\": \"ConnectedValue\"}, \"input_opts\": {\"linefilters\": [{\"__index__\": 0, \"filter\": {\"filter_type\": \"skip\", \"__current_case__\": 0, \"skip_lines\": \"1\"}}]}, \"tbl_opts\": {\"table_name\": \"go\", \"column_names_from_first_line\": false, \"col_names\": \"aspect,go_id,description\", \"load_named_columns\": false, \"pkey_autoincr\": \"\", \"indexes\": [{\"__index__\": 0, \"unique\": false, \"index_columns\": \"aspect,go_id\"}]}}, {\"__index__\": 1, \"table\": {\"__class__\": \"ConnectedValue\"}, \"input_opts\": {\"linefilters\": [{\"__index__\": 0, \"filter\": {\"filter_type\": \"skip\", \"__current_case__\": 0, \"skip_lines\": \"1\"}}, {\"__index__\": 1, \"filter\": {\"filter_type\": \"prepend_line_num\", \"__current_case__\": 2}}]}, \"tbl_opts\": {\"table_name\": \"bering_prot\", \"column_names_from_first_line\": false, \"col_names\": \"id,peptide,uniprot_id,taxon_id,taxon_name,ec_references,go_references,refseq_ids,refseq_protein_ids,insdc_ids,insdc_protein_ids\", \"load_named_columns\": false, \"pkey_autoincr\": \"\", \"indexes\": [{\"__index__\": 0, \"unique\": false, \"index_columns\": \"id,peptide\"}]}}, {\"__index__\": 2, \"table\": {\"__class__\": \"ConnectedValue\"}, \"input_opts\": {\"linefilters\": [{\"__index__\": 0, \"filter\": {\"filter_type\": \"skip\", \"__current_case__\": 0, \"skip_lines\": \"1\"}}, {\"__index__\": 1, \"filter\": {\"filter_type\": \"prepend_line_num\", \"__current_case__\": 2}}, {\"__index__\": 2, \"filter\": {\"filter_type\": \"select_columns\", \"__current_case__\": 7, \"columns\": \"1,7\"}}, {\"__index__\": 3, \"filter\": {\"filter_type\": \"normalize\", \"__current_case__\": 9, \"columns\": \"2\", \"separator\": \" \"}}]}, \"tbl_opts\": {\"table_name\": \"bering_prot_go\", \"column_names_from_first_line\": false, \"col_names\": \"id,go_reference\", \"load_named_columns\": false, \"pkey_autoincr\": \"\", \"indexes\": [{\"__index__\": 0, \"unique\": false, \"index_columns\": \"go_reference,id\"}]}}, {\"__index__\": 3, \"table\": {\"__class__\": \"ConnectedValue\"}, \"input_opts\": {\"linefilters\": [{\"__index__\": 0, \"filter\": {\"filter_type\": \"regex\", \"__current_case__\": 6, \"regex_pattern\": \"^\\\\d\", \"regex_action\": \"include_match\"}}, {\"__index__\": 1, \"filter\": {\"filter_type\": \"select_columns\", \"__current_case__\": 7, \"columns\": \"1,3,23,24\"}}]}, \"tbl_opts\": {\"table_name\": \"bering_psms\", \"column_names_from_first_line\": false, \"col_names\": \"id,sequence,confidence,validation\", \"load_named_columns\": true, \"pkey_autoincr\": \"\", \"indexes\": [{\"__index__\": 0, \"unique\": false, \"index_columns\": \"sequence,id\"}]}}], \"workdb\": \"workdb.sqlite\", \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Comment:
There are still references to the tutorial dataset in the database table names. Can you remove occurences of "bering" here?

Reply:
Hello @wm75 and @clsiguret , do you think simply removing the occurrences of "bering" would work?

Reply:
Hi I don't know what are the "bering" occurences (it's not my wf) :grimacing: 
Do you have an idea @bebatut ?

Reply:
@clsiguret I guess this wf was custom-made for the dataset given in this paper: https://pubs.acs.org/doi/10.1021/acs.jproteome.6b00239

See the first 3 paragraphs given in the tutorial page: 
https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/metaproteomics/tutorial.html#May_2016

Reply:
@subinamehta have you an idea about that?

Reply:
Bering is the dataset we are using to run the workflow. I am not sure why it is a problem though?